dev/dev-tomcat/src/main/tomcat/shared/classes/alfresco-global.properties

```diff
@@ -116,4 +116,9 @@ share.host=localhost
 share.port=8080
 
 
-transformer.debug.entries=10
\ No newline at end of file
+transformer.debug.entries=10
+
+mimetype.config.dir=dev/dev-acs-amps-overlay/target/dev-instance/tomcat/shared/classes/alfresco/extension/mimetypes
```

Comment:
Added other AGPs that matched the pattern as that would simplify development for them too.